CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ add_library(rmm SHARED
 SET_TARGET_PROPERTIES(rmm PROPERTIES BUILD_RPATH "\$ORIGIN")
 
 target_include_directories(rmm PUBLIC
+  "${CUB_INCLUDE_DIR}"
```

Comment:
This is not necessary, and hence shouldn't be added. rmm does use cub directly, only (if at all) via thrust.

Reply:
That's not true. CUB is a transitive dependency of Thrust. If we don't set the correct include directory, the build would use what's included in the CUDA toolkit, which can be an incorrect version.

Reply:
Yeah, sorry, you're right -- thrust does include cub, and when one actually installs thrust, it'll install cub as part of thrust, so that the right cub will be found. But since here we aren't installing thrust but just using it directly from the source, that's not the case (anymore, I believe). I guess I'd say in this case I'd just add both the thrust and the cub include dirs into a single `THRUST_INCLUDE_DIRS` variable, but that doesn't make an actual difference, so it's not worth arguing over.

Overall, though, I think the way the FetchContent is done is kinda hacky, because instead of relying on thrust's build system, which knows what needs to be put where, we're just saying -- it's header-only, and we know what we want, so we'll just pick up the right include paths directly into the thirdparty source. But that's not really related to this PR.